Tests/KituraTests/TestSwaggerGeneration.swift

```diff
@@ -617,6 +623,15 @@ final class TestSwaggerGeneration: KituraTest, KituraTestSuite {
     }
 
     func pathContentAssertions3(paths: [String: Any]) {
+        // Test empty POST
+        if let path = paths["/me/new"] as? [String: Any] {
+            if let new = path["new"] as? [String: Any] {
```

Comment:
This line generates a compile warning because `new` is not used.  Could change to `let _ =` or maybe there's a neater way to check the type?

Reply:
Fixed! Thanks.